freshmaker/pyxis_gql.py

```diff
@@ -90,7 +90,10 @@ def _get_image_projection(self, include_rpms=True):
         projection = [
             ds.ContainerImage.architecture,
             ds.ContainerImage.brew.select(
+                ds.Brew.package,
```

Comment:
I realized that we have some fields included implicitly in LightBlue result, so when I say return same data as the LightBlue result, it's actually inaccurate, some of these fields are not used by any code at all (e.g. `ds.Brew.completion_date`), so could you remove these new fields in this PR? I'll check the existing code later to confirm whether these fields are used.